src/components/button/button.scss

```diff
@@ -5,11 +5,14 @@
 @import "./../../style-config/buttons";
 
 .carbon-button {
+  align-items: center;
   border: $carbon-button__border--secondary;
   border-radius: $carbon-button__border-radius;
   box-sizing: border-box;
   cursor: pointer;
-  display: inline-block;
+  display: inline-flex;
```

Comment:
@abramin Thanks for the cross browser checking!

Could I have some more details on context and steps to reproduce both of the above?